src/isar/apis/schedule/drive_to.py

```diff
@@ -16,76 +12,54 @@
 from models.geometry.position import Position
 from models.planning.step import DriveToPose
 
-api = Namespace(
-    config.get("api_namespaces", "eqrobot_schedule_namespace"),
-    description="Scheduling operations for the robot",
-)
 
-start_response = api.model(
-    "mission_started", {"mission_started": fields.Boolean, "message": fields.String}
-)
-
-
-@api.route(
-    "/drive-to",
-    doc={
-        "description": "Drives to specified position.",
-        "params": {
-            "x": {"description": "The target x coordinate", "type": "float"},
-            "y": {"description": "The target y coordinate", "type": "float"},
-            "z": {"description": "The target z coordinate", "type": "float"},
-            "orientation": {
-                "description": "The target orientation as a quaternion (x, y, z, w)",
-                "type": "list",
-            },
-        },
-    },
-)
-class DriveTo(Resource):
+class DriveTo:
     @inject
     def __init__(self, scheduling_utilities: SchedulingUtilities, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.logger = logging.getLogger("api")
         self.scheduling_utilities = scheduling_utilities
```

Comment:
Note that the flask-restx implementation we used previously had a class `DriveTo(Resource)` which inherited from the `Resource` object. As this is no longer done it is not necessary to call `super().__init__()`. This means `*args` and `**kwargs` may also be removed from the parameters.

Reply:
For reference, `super().__init__(*args, **kwargs)` calles the corresponding `__init__` function for the parent object. The new DriveTo class has no parent.